os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_server.c

```diff
@@ -237,43 +237,43 @@ da_ble_result_type rtw_ble_server_charact_notify(da_ble_attr_handle attr_handle,
         os_mem_free(param);
```

Comment:
Is it okay to free the `param` first?
I think it should be like below.
```
os_mem_free(param->data);
os_mem_free(param);
```

Reply:
@jeongchanKim 
Same as above.
I will check it with the owner of these code in next PR

Reply:
@jeongchanKim I think it's ok because param->date is NULL ,which means memory allocation failed.